src/answers-umd.js

```diff
@@ -303,9 +304,17 @@ class Answers {
     const asyncDeps = this._loadAsyncDependencies(parsedConfig);
     return asyncDeps.finally(() => {
       this._onReady();
+
       if (!this.components.getActiveComponent(SearchComponent.type)) {
         this._initQueryUpdateListener(parsedConfig.search);
       }
+
+      const impressionEvent = createImpressionEvent({
```

Comment:
Here, do we need to look at what components are on the actual page first? If there is no `SearchBar` component on the page, do we fire an impression event? It's also possible that someone could pull down the full SDK and only instantiate a `SearchBar` component. In that case, should we send `standAlone` is true?

Reply:
Those are good points. I'll make some updates

Reply:
Updated. I'm assuming that the standalone assets will always put the search bar on the page, so those assets will always set `standalone: true`